### PR TITLE
Add export savedmodel to wide_deep

### PR DESCRIPTION
**Pull request by k-w-w**
Looks like wide deep is the only official model that doesn't currently export savedmodel. Here are the changes to export it.

### CODE REVIEW

File: official/wide_deep/README.md [hunk 1]
```diff
@@ -47,6 +47,37 @@ Run TensorBoard to inspect the details about the graph and training progression.
...
+```
+
+### Inference
+Let's use the model to predict the income group of two examples:
+```
+saved_model_cli run --dir /tmp/wide_deep_saved_model/${TIMESTAMP}/ \
+--tag_set serve --signature_def="predict" \
+--input_examples='examples=[{"age":[46.], "education_num":[10.], "capital_gain":[7688.], "capital_loss":[0.], "hours_per_week":[38.]}, {"age":[24.], "education_num":[13.], "capital_gain":[0.], "capital_loss":[0.], "hours_per_week":[50.]}]'
+```
+
```
Comment (karmel) [inline official/wide_deep/README.md]: This is great, thanks.

File: official/wide_deep/wide_deep.py [hunk 2]
```diff
@@ -216,6 +216,20 @@ def eval_input_fn():
         flags.stop_threshold, results['accuracy']):
       break
 
+  # Export the model
+  if flags.export_dir is not None:
+    wide_columns, deep_columns = build_model_columns()
```
Comment (karmel) [inline official/wide_deep/wide_deep.py]: Maybe separate out into a separate function to keep the main a little cleaner? Then just call the fn if export_dir is not None.

File: official/utils/arg_parsers/parsers.py [hunk 3]
```diff
@@ -176,6 +177,15 @@ def __init__(self, add_help=False, data_dir=True, model_dir=True,
...
       )
 
+    if export_dir:
+      self.add_argument(
+          "--export_dir", "-ed",
+          help="[default: %(default)s] If set, a SavedModel serialization of "
+               "the model will be exported to this directory at the end of "
+               "training. See the README for more details and relevant links.",
+          metavar="<ED>"
+      )
```
Comment (yhliang2018) [inline official/utils/arg_parsers/parsers.py]: Should we provide a default directory if user provides none? Not sure if this feature is used frequently by users. If so, I think a default dir may be good.

Comment (k-w-w) [reply official/utils/arg_parsers/parsers.py]: Considering that the functionality of SavedModel is currently limited to serving, I don't think this feature is frequently used.

**Comment by k-w-w** [comment]
Will keep in mind in the future -- thanks!

**Comment by vicdxxx** [comment]
Hi, guys! I have tried this command several times:
saved_model_cli run --dir /My Directory/ --tag_set serve --signature_def="predict" --input_examples='examples=[{"age":[46.], "education_num":[10.], "capital_gain":[7688.], "capital_loss":[0.], "hours_per_week":[38.]}, {"age":[24.], "education_num":[13.], "capital_gain":[0.], "capital_loss":[0.], "hours_per_week":[50.]}]' 
But it is always wrong with an error **NameError: name 'age' is not defined**
And if I delete "age", then raise **NameError: name 'education_num' is not defined**
It seems none of keys make sense. and I read the source code, which do call build_model_columns() function and tf.feature_column.numeric_column('age') and so on.
I use windows10 and tensorflow1.8 gpu edition. 
Is this a bug or any thing I configure wrong?
Any suggestion is appreciated!

**Comment by k-w-w** [comment]
@vicdxxx I believe this is due to Windows treating single quotes as part of the input. Try running this:

`saved_model_cli run --dir /My Directory/ --tag_set serve --signature_def="predict" --input_examples="examples=[{'age':[46.], 'education_num':[10.], 'capital_gain':[7688.], 'capital_loss':[0.], 'hours_per_week':[38.]}, {'age':[24.], 'education_num':[13.], 'capital_gain':[0.], 'capital_loss':[0.], 'hours_per_week':[50.]}]"`.

For future questions, please create a [new issue](https://github.com/tensorflow/models/issues) or create a Stackoverflow question (with the tensorflow tag)

**Comment by Menes00** [comment]
Hello, I need to install tensorflow-gpu 1.8 for a project. I have a Geforce GTX 1650ti graphics card. Can you show me a way to do this? @vicdxxx

**Pull request merged**